spark-bigquery-pushdown/pushdown_common_src/main/scala/com/google/cloud/spark/bigquery/pushdowns/SparkExpressionConverter.scala

```diff
@@ -263,7 +263,37 @@ trait SparkExpressionConverter {
               blockStatement(convertStatement(child, fields) + "AS" + cast)
           case _ => convertStatement(child, fields)
         }
-
+      case ShiftLeft(child, position) =>
+        convertStatement(child, fields) + ConstantString("<<") + convertStatement(position, fields)
+      case ShiftRight(child, position) =>
+        convertStatement(child, fields) + ConstantString(">>") + convertStatement(position, fields)
+      case CaseWhen(branches, elseValue) =>
+        ConstantString("CASE") +
+          makeStatement(branches.map(whenClauseTuple =>
+            ConstantString("WHEN") + convertStatement(whenClauseTuple._1, fields) + ConstantString("THEN") + convertStatement(whenClauseTuple._2, fields)
+          ), "") +
```

Comment:
Shouldn't the delimiter be a space here rather than ""?

Reply:
There is a space already added between two WHEN's because of the `convertStatement`. The unit test added for this case has the space too. That is why I have not added any space for the delimiter.